Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -58,8 +58,8 @@ class WelcomeViewController: BaseViewController {
         })
         alert.addAction(UIAlertAction(title: NSLocalizedString("Ok", comment:""), style: .default, handler: {[weak alert] (action) -> Void in
             let textField = alert!.textFields![0] as UITextField
-            let strURL = textField.text?.replacingOccurrences(of: " ", with: "")
```

Comment:
Good to extract this to a method on its own!

Reply:
OK. Done